### PR TITLE
Patch for RPM generation on SLES

### PR DESCRIPTION
**Pull request by rafalste**
Signed-off-by: Rafal Stefanowski <rafal.stefanowski@intel.com>

### CODE REVIEW

File: utils/pckgen [hunk 1]
```diff
@@ -257,25 +258,34 @@ generate_rpm() {
     rpm_obtain_sources
     rpm_spec_prepare
 
+    . $SYS_INFO
+    if [[ "$ID_LIKE" =~ suse|sles ]] && [ -d /usr/src/packages ]; then
+        info "INFO: It appears that you are using SUSE Linux."\
```
Comment (Ostrokrzew) [inline utils/pckgen:263]: missing 'if' probably

Comment (rafalste) [reply utils/pckgen:263]: Everything till the semicolon is handled by the single if statement.

Comment (Ostrokrzew) [reply utils/pckgen:263]: I mean word 'if' in info.

Comment (rafalste) [reply utils/pckgen:263]: I think the info message is correct. :)

**Review by Ostrokrzew — APPROVED** [review]

**Review by arutk — APPROVED** [review]

**Pull request merged**